ios/TwilioVoiceReactNative.m

```diff
@@ -439,6 +441,12 @@ - (void)sendEventWithName:(NSString *)eventName body:(id)body {
         self.deviceTokenData = [testDeviceToken dataUsingEncoding:NSUTF8StringEncoding];
     }
 #endif
+    
+    if (!self.deviceTokenData) {
```

Comment:
This may happen when the app calls `voice.register()` too soon when the device token is still not available. Is it possible that we wait for the device token here and continue registering when the value is available?